datascience/README.md

```diff
@@ -52,3 +59,41 @@ Returns :
 3      DE      4229.0  biomass  2016-12-23T00:00:00+00:00
 4      DE     19919.0     coal  2016-12-23T00:00:00+00:00
 ```
+
+#### get_exchange(countries, start_date, end_date)
+
+This function returns for each day between `start_date` and `end_date`, the electricity production exchanges for each country defined in `countries`. Data are returned as a [pandas DataFrame](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html).
+
+Parameters :
+
+- countries (*list*) - a list of country codes to pull.
```

Comment:
Ok makes sense. We should make it clear though. Maybe instead of saying "a list of country codes to pull" we could say "a list of country codes for which to pull exchanges with neighbouring countries."?